index.css

```diff
@@ -0,0 +1,201 @@
+* {
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+}
+
+html,
+body {
+    height: 100%;
+    font-size: 14px;
+    font-family: arial, sans-serif;
+}
+
+.main {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    width: 100%;
+    background-color: #fff;
+    position: relative
+}
+
+.navbar {
+    display: flex;
+    flex-direction: row-reverse;
+    padding-top: 20px;
+    height: 8%
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    align-items: center;
+}
+
+li {
+    display: inline;
+    margin-right: 20px;
+}
+
+a {
+    text-decoration: none;
+    color: #000000;
+}
+
+img {
+    height: 24px;
+}
+
```

Comment:
- Always try to reduce the impact area from your changes in the project.
- All of these styles that you have applied are directly impacting the HTML tags.
- Ask these questions to yourself:
   - Do you need white color on every link? If no why have you applied it for all anchor tags.
   - Do you need a margin right on all `li` elements? If no can you try to narrow down the impact of your changes?
   - You can ask same question for all the global changes that you're making. **THINK HOW YOUR CHANGES ARE GOING TO IMPACT THE PROJECT?**
- It is always a good idea to narrow down the impact of your changes as much as possible.
- Instead of overwriting these tags directly, you could have created classes for them or you could possibly target these tags under a certain class.
- Example:
```
<style>
// You bsaically narrowed down the impact of your code 
// to certain anchor tags that only live // inside a div that have class `myClass` applied on it.

.myClass a {
 // your styles
}

</style>


<div class="myClass">
      <a href='#'>ABC</a>
<div>
```